extensions/amp-date-picker/0.1/date-range-picker.js

```diff
@@ -108,6 +109,13 @@ function createDateRangePickerBase() {
     displayFormat: () => moment.localeData().longDateFormat('L'),
     monthFormat: 'MMMM YYYY',
 
+    // TODO(cvializ): make these configurable for i18n
```

Comment:
Why isn't this a `map` from `utils/object.js`?

Reply:
It should be a `map`. I'll make a cleanup pass and use `map`s, and I should do vsync mutates and measures where applicable too.